js/compiler.ts

```diff
@@ -250,10 +251,6 @@ export class DenoCompiler implements ts.LanguageServiceHost {
       }
       const dependencyMetaData = this._getModuleMetaData(dep);
       assert(dependencyMetaData != null, `Missing dependency "${dep}".`);
-      assert(
```

Comment:
This was, in the end, a _bad_ assertion, because with circular dependencies, you will have some modules that are not yet run, but we will be returning the reference to their `.exports` object to a module.